recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export DYLD_LIBRARY_PATH=$PREFIX/lib
+
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+export LD_LIBRARY_PATH=$PREFIX/lib
+
+./configure \
```

Comment:
Are we bound to use `configure` here? I had a look at [how unidata is testing](https://travis-ci.org/Unidata/netcdf-c/jobs/118246341). They also use a special docker image, but use cmake instead of configure.